setup.cfg

```diff
@@ -53,6 +53,113 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
+    pytest-asyncio>=0.17.0
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+performance =
+    bottleneck>=1.3.2
+    numba>=0.53.0
+    numexpr>=2.7.1
+timezone =
+    tzdata>=2022.1
+computation =
+    scipy>=1.7.1
+    xarray>=0.19.0
+fss =
```

Comment:
I would personally like to see this be called `filesystem` as `fss` isn't immediately obvious to me, but not a blocker to me.

Reply:
It's a good question.  I went back and forth on `fss` vs `filesystem`.  Ultimately I decided that `filesystem` might suggest to users a broad compatibility with common filesystems (eg. s3) that the optional_extra does not provide, and it was probably least surprising to users to call it `fss`. (and the alternative `fsspec` is worryingly specific in case fsspec were ever deprecated and pandas switched to an alternative filesystem mgmt package in future).

But I don't think either solution is perfect and am very open to alternative suggestions or being argued back to `filesystem`.